source/devel_docker_ci.rst

```diff
@@ -0,0 +1,251 @@
+=============================
+Docker CI with GitHub Actions
+=============================
+
+.. _Docker Hub: https://hub.docker.com/
+.. _Docker: https://www.docker.com/
+.. _GitHub: https://github.com/
+.. _CI: https://en.wikipedia.org/wiki/Continuous_integration
+.. _GitHub Actions Documentation: https://help.github.com/en/actions
+.. _GitHub Actions: https://github.com/features/actions
+.. _Overview of Docker Compose: https://docs.docker.com/compose/
+.. _Automated repository tests: https://docs.docker.com/docker-hub/builds/automated-testing/
+.. _semver.org: https://semver.org/
```

Comment:
Could you change to: `Semantic Versioning: https://semver.org/`?

Reply:
I'll fix the link further down in the documentation and update.

Reply:
Updating to Semantic Versioning 2.0.0 since it's the title on the web-page.